docs/design/rules-api.md

```diff
@@ -0,0 +1,111 @@
+# Rules API
+
+Observatorium is featured with a multi-tenant API that enables tenants to write and read Prometheus recording and alerting rules.
+
+The API is [defined](https://github.com/observatorium/api/blob/main/rules/spec.yaml) using the [OpenAPI](https://swagger.io/specification/) specification format.
+
+The goal is to enable tenants to create, modify and access their own rules.
+
+## Usage
+
+### Create rules
+
+```bash
+PUT /api/v1/rules/raw
+```
+
+Set rules for a tenant.
+
+#### Example request
+
+```bash
+curl -X PUT --data-binary @alerting-rule.yaml --header "Content-Type: application/yaml" http://<observatorium-api-url>/api/metrics/v1/<tenant>/api/v1/rules/raw
+```
+
+Where:
+
+* `rule.yaml` is a YAML file containing the desired rule definition. It can contain recording rules, alerting rules or both.
+  * *Note: For every `PUT` request, the content of the YAML file will be overwritten with the current content being sent.*
```

Comment:
I'm worried this sentence may be a bit confusing for readers, but I'm not sure how to rephrase it.

Maybe:

Each time a PUT request is made, the rules contained in the request will overwrite all other rules for that tenant.